index.html

```diff
@@ -6049,9 +6090,15 @@ <h3>Executing Script</h3>
   <a>getting a property</a> named <code>script</code>
   from the <var>parameters</var>.
 
+ <li><p>If <var>script</var> is set and <var>name</var> is set,
+ return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
  <li><p>If <var>script</var> is not a <a>String</a>,
   return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
 
+ <li><p>If <var>name</var> is set let <var>script</var> be the result
+ of <a>trying</a> to <a>get a script</a> with argument <var>name</var>
```

Comment:
This is nitpicky, but I would prefer the slightly more English:

s/with argument/by/